### PR TITLE
Execution/gunpoint shots work properly with pellet clouds, buffs piercing wounds a bit

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Made a few changes to gunpoints and some related mechanics while tidying up and adding docs. Here's a quick list

- Execution (point blank help intent shot aimed at the mouth) and gunpoint shots now apply their bonuses to every pellet fired rather than only the first, generally making them gushier. They also buff wound power as well as damage
- You can no longer punch yourself while holding someone up to trigger the charged shot
- You can no longer purposely fail executions and gunpoints with pax or whatever to endlessly spike the damage in a loaded round
- Attacks with extremely high wounding power can now outright dismember limbs regardless of mangled status. The threshold is high enough that it mostly applies for admin edited weapons or execution shots with shotguns (or people with the frail quirk!)
- Piercing wounds make further wounds a bit easier to apply to give them a bit more power
- Hellguns now cost 2250 credits instead of 2000 to make them a bit harder for random crew to get
- Adds special bouncy L6 rounds for admins to use to bounce off anything and everything en-masse

Also as a minor note
- Projectiles with no trajectory (meaning they were likely spawned in manually) now qdel themselves on bumping something, rather than runtiming helplessly

## Why It's Good For The Game
Makes shotgun executions and gunpoints gushier and more satisfying, lets admins create weapons capable of outright eradicating peoples limbs with VV, counters gunpoint self punching cheese. Also the spawned projectiles runtiming when ran into was annoying as an admin, so it no longer happens

The outright dismemberment was something that was on my mind since the changes in Wounds 2 made it so you always needed at least 2 hits to break the flesh and bone on a limb before dismembering. This lets admins create weapons that can do it in one with an absurdly high wound_bonus value, as well as if someone is absurdly messed up but doesn't meet the mangling requirements somehow

## Changelog
:cl: Ryll/Shaps
tweak: Execution and gunpoint shots now increase the wounding power of the shot as well as the damage, and apply their bonuses to each pellet fired rather than just one
tweak: Piercing wounds make further wounds slightly easier to deal
balance: Hellguns now cost 2250 credits instead of 2000
fix: You can no longer punch yourself while holding someone up to trigger the charged shot
fix: You can no longer purposely fail executions or gunpoints to spike the damage in a loaded round for later
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Merge conflict.
Let us know when it's fixed if it's properly tested locally.
If so, this looks ready to go.

File: code/datums/components/gunpoint.dm [hunk 1]
```diff
@@ -113,6 +117,8 @@
 
 ///Bang bang, we're firing a charged shot off
 /datum/component/gunpoint/proc/trigger_reaction()
+	SIGNAL_HANDLER_DOES_SLEEP
```
Comment (Jared-Fogle) [inline code/datums/components/gunpoint.dm:120]: This shouldn't be used on anything new. See if you can move the async parts to their own proc and call that asynchronously.

Comment (Rohesie) [reply code/datums/components/gunpoint.dm:120]: Just for clarity, the PR is not adding this signal.

Comment (Jared-Fogle) [reply code/datums/components/gunpoint.dm:120]: Ah, nevermind then. I must have just missed it when I did my PR.

**Comment by Rohesie** [comment]
This looks ready. Let us know when you feel it is.

**Pull request merged**